.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,24 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/cpp
 {
-	"name": "C++",
+	"name": "R-Dev-Env",
 	"build": {
 		"dockerfile": "Dockerfile"
+         },
+	 "hostRequirements": {
+	  "cpus": 4
 	},
 	"customizations": {
 		"vscode": {
 			"extensions": [
-				"REditorSupport.r"
+				"REditorSupport.r",
+				"mads-hartmann.bash-ide-vscode",
+				"quarto.quarto",
+				"johnstoncode.svn-scm"
 			]
 		}
-	}
+	},
+	"postStartCommand": "bash ./scripts/setup.sh"
```

Comment:
```suggestion
```
The environment variables are defined in the container so it might be easier to not include this quite yet. We can put this into another branch or have this as a script the user can run if they like (if they are looking at downloading multiple versions of R, for instance).